public/app/features/admin/UserOrgs.tsx

```diff
@@ -174,7 +164,7 @@ class UnThemedOrgRow extends PureComponent<OrgRowProps, OrgRowState> {
                 onCancel={this.onCancelClick}
                 onConfirm={this.onOrgRemove}
               >
-                Remove from organisation
```

Comment:
We decided to have text here when have been working on page design with @sarlinska

Reply:
Gotcha, I'll just use the link version of the button, so it isn't so much `INYOURFACE`